bin/near

```diff
@@ -80,7 +80,19 @@ const createAccount = {
       type: 'string',
       required: true
     }),
-  handler: (argv) => exitOnError(main.createDevAccount(argv))
+  handler: (argv) => exitOnError(main.createAccount(argv))
+};
+
+const viewAccount = {
```

Comment:
https://github.com/nearprotocol/near-shell/commit/bf53541192855b03e71853dc00c4b6d5f28163ef

Reply:
Btw, I'm fine with `view_account` instead of `state` command. You tell me what do you think is better.

Reply:
`state` is shorter and makes sense